### ISSUE TITLE
Set favicon for GoAccess?

### ISSUE THREAD

**Issue by jockebq** [allinurl/goaccess#1561]
How do I set favicon for GoAccess?
Placing a favicon.ico in the root does nothing?

Thank you!

**Comment by allinurl** [comment]
Please take a look at this [**post**](https://stackoverflow.com/questions/9943771/adding-a-favicon-to-a-static-html-page).

**Comment by jockebq** [comment]
> Please take a look at this [**post**](https://stackoverflow.com/questions/9943771/adding-a-favicon-to-a-static-html-page).

But won't that be overwritten all the time? I have GoAccess running every 10 minutes in crontab to output a html file in my webserver.

**Comment by allinurl** [comment]
You're right. Then the easiest is to add the line [here](https://github.com/allinurl/goaccess/blob/master/src/output.c#L222) and [build](https://goaccess.io/download#installation) goaccess again. Otherwise you can try using a custom snippet in JS using the `html-custom-js /path/file.js`
